### PR TITLE
added a commented description to the has_many/has_one :through example of Schema

### PR DESCRIPTION
**Pull request by elbow-jason**
After some confusion about has_many/has_one :through I was requested to improve the has_many/has_one :through documentation. Hopefully the comment will prevent other people from having the same issue.

### CODE REVIEW

File: lib/ecto/schema.ex [hunk 1]
```diff
@@ -478,6 +478,13 @@ defmodule Ecto.Schema do
           has_one :permalink, Permalink
           has_many :comments_authors, through: [:comments, :author]
 
+          # From the has_many :through example above, in the list
```
Comment (josevalim) [inline lib/ecto/schema.ex]: Can we move this comment to right above the `has_many :through` thing?

Comment (elbow-jason) [inline lib/ecto/schema.ex]: will do

**Comment by elbow-jason** [comment]
I moved the comment up and corrected the wording

**Comment by josevalim** [comment]
Merged manually, thank you

**Comment by elbow-jason** [comment]
:+1:

**Pull request closed without merging**